docs/integrations/mirroring.md

```diff
@@ -0,0 +1,101 @@
+
+An example Mirroring Integration can be seen [here](https://github.com/demisto/content/tree/master/Packs/XSOARMirroring/Integrations/XSOARMirroring).
+
+Mirroring Integrations are developed the same as other Integrations. They provide a few extra configuration parameters and APIs.
+
+
+## Supported Server Version
+Mirroring Integration's YAML file _must_ have the following field `fromversion: 6.0.0`. This is because Mirroring Integrations are only supported from Server version 6.0.0 and onwards.
+
+
+## Required Parameters
+Mirroring integration should have the following parameters in the integration YAML file(under the scripts section):
+```yml
+  isfetch: true
+  ismappable: true
+  isremotesyncin: true
+  isremotesyncout: true
+```
+While the `ismappable` and `isremotesyncin` are used for getting information from a remote incident, And the `isremotesyncout` for updating remote data.
+
+## Commands
+Use the following commands to implement a mirroring integration.  
+*Note that when mirroring both incoming and outgoing data all the commands are required, while only some of the commands are required for a one-way mirror.*
+- `test-module` - this is the command that is run when the `Test` button in the configuration panel of an integration is clicked.
+- `fetch-incidents` - this is the command that fetches new incidents to Cortex XSOAR.
+- `get-remote-data` - this command takes new information about the incidents in the remote system and updates the *already existing* incidents in Cortex XSOAR. This command is being executed every 1 minute. 
+- `update-remote-system` - this command updates the remote system with the information we have in the mirrored incidents within Cortex XSOAR. This command is being executed every 1 minute.
+- `get-mapping-fields` - this command will pull the different incident types and their associated incident fields from the remote system, it will be used in the Mirror out configuration page in the XSOAR UI.
+
+## Useful objects to use in your code
+All the functions explained here are globally available through the CommonServerPython file.
+
+### get-remote-data
+* GetRemoteDateArgs - this is an object created to maintain all the argument you receive from the server in order to use this command.
+Arguments explanation:
+  - remote_incident_id - the remote incident id.
+  - last_update - the time the incident was last updated - our recommendation would be to use the arg_to_timestamp function to parse it into a timestamp.
+* GetRemoteDataResponse - this is the object that maintains the format in which you should order the results from this function. You should use return_results on this object to make it work.
+Arguments explanation:
+  - mirrored_object - this is essentially the object(dict) of whatever you are trying to mirror - incident in most cases.
+  - entries - a list of entries to add to your incident - for instance:
+```buildoutcfg
+{
+    'Type': entry.get('type'),
```

Comment:
this is example is not clear at all.
Provide full example of `get-remote-data` with hard coded values. Don't use constants from XSOAR mirroring, because the reader don't see the full code.
Instead set hard coded values, this way the reader will be able to know what kind/types he should use/pass/return